conf/messages

```diff
@@ -510,34 +510,48 @@ numberOfProperties.error.invalid=Select how many UK properties you are including
 numberOfProperties.error.required=Select how many UK properties you are including in this return
 
 disposalMethod.title=How did you sell or dispose of the UK property?
+disposalMethod.agent.title=How did your client sell or dispose of the UK property?
 disposalMethod.Sold=Sold it
 disposalMethod.Gifted=Gave it away
 disposalMethod.Other=Other
-disposalMethod.error.required=Select if you sold it, gave it away or other
-disposalMethod.error.invalid=Select if you sold it, gave it away or other
+disposalMethod.error.required=Select sold it, gave it away or other
+disposalMethod.error.invalid=Select sold it, gave it away or other
 
 wereYouAUKResident.title=Were you a UK resident when you exchanged contracts when disposing of the property?
-wereYouAUKResident.helpText=Contract exchange usually happens when you and the buyer or new owner swap contracts and the buyer pays a deposit.
-wereYouAUKResident.link=<a href="{0}" target="_blank">Work out your residence status for a tax year (opens in a new window)</a>.
+wereYouAUKResident.agent.title=Was your client a UK resident on the day contracts were exchanged?
+wereYouAUKResident.helpText=Contract exchange usually happens when a buyer and seller swap contracts and the buyer pays a deposit.
+wereYouAUKResident.link=<a href="{0}" target="_blank">Work out your residence status for a tax year (opens in a new window)</a>
```

Comment:
should this and the message below it have a full stop?

Reply:
links should not have full stops - only when they are the end of sentence then we see the sentence full stop

Reply:
this link is a standalone link